doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,598 @@
+A good way of knowing what CSCs are part of the group you are working with is to print this list;
+
+.. code:: python
+
+    print(atcs.components)
+
+It is also possible to use this list to access the remotes programmatically, e.g.;
+
+.. code:: python
+
+    # Get one heartbeat and print the state of each component in ATCS
+    for comp in atcs.components:
+      await getattr(atcs.rem, comp).evt_heartbeat.next(flush=True, timeout=5)
+      comp_state = await atcs.get_state(comp)
+      print(f"{comp}: {comp_state!r}")
+
+
+.. _user-guide-generic-telescope-control-operations:
+
+Generic Telescope Control System Operations
+===========================================
+
+The same way groups of CSCs contains generic operations, a group of CSCs that is part of a Telescope Control System (TCS) group retain some common operations.
+For instance, a TCS will be responsible for slewing the telescope and tracking a target on the sky.
+The same way, a TCS will also be responsible for preparing the telescope for calibrations and on-sky operations.
+
+To model these common TCS behavior the package implements :py:class:`BaseTCS <lsst.ts.observatory.control.BaseTCS>`.
+This class itself is an `abstract class <https://docs.python.org/3/library/abc.html>`__ and cannot be used stand-alone, but is fully implemented by :py:class:`ATCS <lsst.ts.observatory.control.auxtel.ATCS>` and :py:class:`MTCS <lsst.ts.observatory.control.maintel.MTCS>`.
+There are many advantages of this implementation as it minimizes code duplication and also provides users with a common interface and feature set, regardless of what telescope they are working with.
+
+In terms of setting up and shutting down the system :py:class:`BaseTCS <lsst.ts.observatory.control.BaseTCS>` expands to add the following methods:
+
+  * :py:meth:`prepare_for_flatfield <lsst.ts.observatory.control.BaseTCS.prepare_for_flatfield>`, to prepare the system for calibrations.
+  * :py:meth:`prepare_for_onsky <lsst.ts.observatory.control.BaseTCS.prepare_for_onsky>`, to prepare the system for on-sky operations.
+  * :py:meth:`shutdown <lsst.ts.observatory.control.BaseTCS.shutdown>` to stow the system.
+
+The actual operation performed by each of those tasks is particular to the final implementation and is detailed in :ref:`user-guide-atcs` and :ref:`user-guide-mtcs`.
+
+Furthermore, the TCS provides a suit of useful slew operations.
+
+To slew the telescope to an AzEl coordinate (for instance, to conduct some maintenance of calibration), it is possible to use :py:meth:`point_azel <lsst.ts.observatory.control.BaseTCS.point_azel>`.
+The method will slew to a fixed position in the local coordinate and `will not` initiate tracking.
+For instance, to slew the telescope to `azimuth=0.` and `elevation=80.` degrees;
+
+.. code:: python
+
+    await tcs.point_azel(az = 0., el=80.)
```

Comment:
so I think this might be a bit confusing. Maybe you want to show the instantiation of the tcs class showing which tcs you're using? Otherwise it sounds a bit like you're using the baseclass to do this.

Reply:
I added a note before about that... hope it helps ease the confusion.